client/reader/search-stream/index.jsx

```diff
@@ -27,6 +27,10 @@ import Suggestion from './suggestion';
 import ReaderPostCard from 'blocks/reader-post-card';
 import { RelatedPostCard } from 'blocks/reader-related-card-v2';
 import config from 'config';
+import {
+	EMPTY_SEARCH_RECOMMENDATIONS,
+	SEARCH_RESULTS,
+ } from 'reader/follow-button/constants';
```

Comment:
Sorry @samouri - poor filename suggestion on my part. Shall we call the file `follow-sources.js` instead? (Consistent with `action-types.js` in Redux then.)